CONTRIBUTERS

```diff
@@ -0,0 +1 @@
+Ryan Artecona <ryanartecona@gmail.com>
```

Comment:
I think the correct spelling is "contributors" for the file name

Reply:
corrected